Sources/PT.PM.PythonParseTreeUst/PythonAntlrConverter.cs

```diff
@@ -984,12 +992,20 @@ public Ust VisitEncoding_decl(PythonParser.Encoding_declContext context)
 
         public Ust VisitYield_expr(PythonParser.Yield_exprContext context)
         {
-            return VisitChildren(context);
+            var yielArg = context.yield_arg();
+            return yielArg == null
+                ? new YieldExpression(null, context.GetTextSpan())
+                : new YieldExpression(Visit(context.yield_arg()).ToExpressionIfRequired(), context.GetTextSpan());
```

Comment:
I think you can reuse `yielArg` here. And rename it to `yieldArg` by the way. :)